index.html

```diff
@@ -9,184 +9,193 @@
+
+            <div class="languages">Languages <i class="fa-solid fa-angle-down"></i>
+              <div  class="languages"><img src="images/javascript.png" alt="Javascript">
+              <p>Javascript</p></div>
+              <div  class="languages"><img src="images/html.png" alt="HTML">
+               <p>HTML5</p></div>
+              <div  class="languages"><img src="images/css.png" alt="CSS">
+                 <p>CSS3</p></div>
+              </div>
+
+
+
+        
+        <div class="languages">Frameworks<i class="fa-solid fa-angle-down"></i>
+            
         </div>
-        <section>
-            <ul class="lists">
-                <li class="layout collapse">
-                    <div class="textlayout">Languages</div><img class="vector" src="vector1.png" alt="vector">
-                </li>
-                <li class="layout frame"><img class="pad" src="javascript.png" alt="js">
-                    <div class="textLi">Javascript</div>
-                </li>
-                <li class="layout frame"><img class="pad" src="html.png" alt="html">
-                    <div class="textLi">HTML</div>
-                </li>
-                <li class="layout frame"><img class="pad" src="css.png" alt="css">
-                    <div class="textLi">CSS</div>
-                </li>
-            </ul>
-        </section>
-        <section>
-            <ul>
-                <li class="layout collapse">
-                    <div class="textlayout">Frameworks</div><img class="vector1" src="vector1.png" alt="vector">
-                </li>
-            </ul>
-        </section>
-        <section>
-            <ul>
-                <li class="layout collapse">
-                    <div class="textlayout">Skills</div><img class="vector2" src="vector1.png" alt="vector">
-                </li>
-            </ul>
-        </section>
-        <section>
 
-        </section>
+
+        <div class="skills">Skills<i class="fa-solid fa-angle-down"></i>
+            
+        </div>
+
+    </section>
+
 
 
+    <section class="contact-me">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that
+            needs coding. I'd love to help with it </p>
 
+        <form action="https://formspree.io/f/xeqngele" method="post">
+            <ul>
+                <li>
+                    <input type="text" name="username" placeholder="Enter your username" maxlength="30" required>
+                </li>
+                <li>
+                    <input type="email" name="email" placeholder="Enter your email address" required>
+                </li>
+                <li>
+                    <textarea placeholder="Write a message here" maxlength="500" required></textarea>
+                </li>
+            </ul>
+            <button type="submit" id="submit-button">Get in Touch</button>
+        </form>
+        <img src="images/Shape.png" alt="homeindicator">
```

Comment:
- [ ] Please, kindly remove this image, as it is not part of the page design. In the Figma template, it is a home indicator icon that is only representing mobile screen elements.